lib/modules/datasource/go/base.ts

```diff
@@ -71,6 +71,24 @@ export class BaseGoDatasource {
       };
     }
 
+    if (goModule.startsWith('dev.azure.com/')) {
+      const split = goModule.split('/');
```

Comment:
Can you do a sanity check to make sure the split length is what you expect?

Reply:
Add sanity check with latest commit